content/soda-gettingstarted/ak-sk-using-api.md

```diff
@@ -0,0 +1,154 @@
+  
+##### SK: secret access key used in conjunction with an AK to sign requests cryptographically. It identifies a request sender and prevents the request from being modified.
+
+{{% notice info %}}
+**_To use the multicloud service, an Access Key, Secret Key (AK/SK) must be generated before anything else. To do this you can follow the steps below:_**  <br />
+Assuming that the user have access to cURL, the API execution is displayed using cURL commands.
+{{% /notice %}}
+
+
+#### 1. Create the Token for the "admin" User.
+API URI Format : 'http://<HOST:PORT>/identity/v3/auth/tokens'
+
+```bash
+curl --location --request POST 'http://192.168.xx.yy/identity/v3/auth/tokens' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "auth": {
+    "identity": {
+      "methods": [
+        "password"
+      ],
+      "password": {
+        "user": {
+          "name": "admin",
+          "domain": {
+            "name": "Default"
+          },
+          "password": "password"
+        }
+      }
+    }
+  }
+}'
+```
+
+Similarly, tokens can be generated for any of the users. To generate token for any specific user, replace the "name": "admin" with the <user-name> of the "user" and the user's corresponding password. 
+
+#### 2. Generate the Scoped Tokens 
+API URI Format : 'http://<HOST:PORT>/identity/v3/auth/tokens'
+
+  ```bash
+  curl --location --request POST 'http://192.168.xx.yy/identity/v3/auth/tokens' \
+  --header 'X-Auth-Token: gAAAAABg1Ybtbnk2NEbyewe_mOILyo5v0PnaTWGoSyMhq75NaWR_L111111111111111111111_I222222222_M3333333333_ICZUHCIDLtU0pIcKGAvN9zJKXaXBQQIf27a3uq2WM8eQroQ-CMc1-riHbCbrs_iFucc2Z4RCGEzxL6YqTvlzBoarjIsx5' \
+  --header 'Content-Type: application/json' \
+  --data-raw '{
+    "auth": {
+      "identity": {
+        "methods": [
+          "token"
+        ],
+        "token": {
+          "id": "gAAAAABg1Ybtbnk2NEbyewe_mOILyo5v0PnaTWGoSyMhq75NaWR_L111111111111111111111_I222222222_M3333333333_ICZUHCIDLtU0pIcKGAvN9zJKXaXBQQIf27a3uq2WM8eQroQ-CMc1-riHbCbrs_iFucc2Z4RCGEzxL6YqTvlzBoarjIsx5"
+        }
+      },
+      "scope": {
+        "project": {
+          "name": "admin",
+          "domain": {
+            "id": "default"
+          }
+        }
+      }
+    }
+  }'
+  ```
+
+Generate the Scoped Tokens for the user. Here, the scoped token for the "admin" user is generated, using the tokens obtained in the Response Header "X-Subject-Token" from Step 1. 
+
+#### 3. Using the token AK/ SK can be generated using the POST API 
+
+  After the Scoped Tokens are generated, ProjecID and UserID can be obtained from them. 
+  For eg, if the ProjectID and UserID obtained are 
+
+  ```bash
+    projectId : "94b280022d0c4401bcf3b0ea858777777" 
+    userId : "555557c4256555bd8a307c374644444" 
+  ```
+
+  then the command would look like this. 
+
+  API URI Format : 'http://<HOST:PORT>/v1/<project_id>/aksks'
```

Comment:
Why <project_id> requied in URI and Body, at two place?

Reply:
It is to maintain a consistent Request Body , request body with just a user_id looks incomplete.